src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlConnector.cs

```diff
@@ -61,9 +63,15 @@ internal ValueTask Close(bool async)
         /// <param name = "cancellationToken">The token used to cancel an ongoing asynchronous call.</param>
         /// <returns>A Task indicating the result of the operation.</returns>
         /// <exception cref="NotImplementedException"></exception>
-        internal ValueTask Open(TimeSpan timeout, bool async, CancellationToken cancellationToken)
+        internal async ValueTask Open(TimeSpan timeout, bool async, CancellationToken cancellationToken)
         {
-            throw new NotImplementedException();
+            HandlerOrchestrator orchestrator = HandlerOrchestrator.Instance;
+            ConnectionHandlerContext context = new ConnectionHandlerContext
```

Comment:
Don't we need to hang on to the hydrated context? Otherwise we'll lose the streams we created.